boards/shields/xenvm_dom0/Kconfig.shield

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2023 EPAM Systems, Inc
+# SPDX-License-Identifier: Apache-2.0
+
+config SHIELD_XENVM_DOM0
+	def_bool $(shields_list_contains,xenvm_dom0)
```

Comment:
Could you please add `depends on ARMV8_A`? I don't believe that this would work on CorteR or MIPS :)